src/Sylius/Behat/Context/Api/Shop/ProductContext.php

```diff
@@ -503,4 +533,74 @@ private function hasProductWithNameAndShortDescription(array $products, string $
 
         return false;
     }
+
+    private function hasProductOptionWithNameAndValue(string $optionValue, string $optionName): bool
+    {
+        $response = $this->client->getLastResponse();
+        $productOptions = $this->responseChecker->getValue($response, 'options');
+
+        foreach($productOptions as $option){
+            if ($this->hasOption($option, $optionName)) {
+                $variants = $this->responseChecker->getValue($response, 'variants');
+
+                foreach($variants as $variant) {
+                    if ($this->variantHasProductOptionValue($variant, $optionValue))
+                    {
+                        return true;
```

Comment:
```suggestion
                foreach ($variants as $variant) {
                    if ($this->variantHasProductOptionValue($variant, $optionValue)) {
                        return true;
```
Similar as above with the `$variant`, change the name of this variable